samples/MusicStore.Spa/Properties/launchSettings.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
Why do we ignore the launch settings? They have to be kept up to date as things change (e.g. the environment variable name for current environment).